src/js/BUILD.md

```diff
@@ -0,0 +1,70 @@
+### Build OpenVINO™ GenAI Node.js bindings (preview)
+
+1. Build and install OpenVINO from sources following the [instructions](https://github.com/openvinotoolkit/openvino/wiki#how-to-build).  
```

Comment:
Is OpenVINO required to be built with JS bindings? I wasn't able to find JS instruction on the wiki. If JS bindings aren't required, it's worth specifying it here anyway to be more verbose.

Reply:
Here we need to build OpenVINO without JS bindings because it is used to build GenAI.
But you are right that the user should use the GenAI JS package from source with the correct version of the OpenVINO JS package.
I've added information about this in the README that says how it should be used.

Reply:
Yes, I have done.